docs/user/getting-started.md

```diff
@@ -222,29 +222,116 @@ Browse to `ui.flightctl.MY.DOMAIN` and login with the demouser obtained from the
+Next, we will use [Podman](https://github.com/containers/podman) to build a [bootable container image (bootc)](https://containers.github.io/bootc/) that includes the Flight Control Agent binary and configuration. The configuration contains the connection details and credentials required by the agent to discover the service and send an enrollment request to the service. After deploying the flightctl server, this configuration can be retrieved after a Certificate Signing Request has been completed as outlined in the steps below.
 
-Retrieve the agent configuration with enrollment credentials by running:
+### Create Certificate Signing Request configuration
+
+For enrollment, the device will need an enrollment certificate to be present. The enrollment certificate is created when a certificate signing request in flightctl is approved by an authorized user and can be retrieved once this process is complete. The four steps below describe this process.
+
+#### 1/4 Create the signed CSR
+
+This CSR will be embedded in the CSR resource configuration file that will be applied to flightctl.
+
+Create a certificate signing request `.csr` file with openssl:
 
 ```console
-$ flightctl certificate request --scope=enrollment --validity=1y -o agent-config > config.yaml
+openssl req -new -sha256 -key myeckey.pem -out mycsr.csr
+```
 
-[...]
+**NOTES**:
+
+1. The signing key passed with `-key` MUST be an ECDSA key. To generate an ECDSA key, use `openssl ecparam -name secp521r1 -genkey -noout -out myeckey.pem` or see the [openssl documentation on ECDSA](https://docs.openssl.org/1.0.2/man1/ecparam/#synopsis) for more options.
+2. The Subject Common Name in the CSR MUST be at least 16 characters in length.
+
+For more options, including generating a new private key to sign the CSR, see [openssl's documentation](https://docs.openssl.org/master/man1/openssl-req/#options).
+
+#### 2/4 Create the CSR resource configuration file
+
+The file name, `metadata.name`, and `spec.expirationSeconds` can vary. The `apiVersion`, `kind`, `spec.signerName`, and `spec.usages` must match those below. The `spec.request` field will hold a base64-encoded contents of the `.csr` file previously created.
+
+```console
+$ cat > mycsrresource.yaml <<EOF
+apiVersion: v1alpha1
+kind: CertificateSigningRequest
+metadata:
+  name: mycsr
+spec:
+  request: <add base64-encoded CSR>                                                                                                                                                                                                             signerName: ca
+  usages: ["clientAuth", "CA:false"]
+  expirationSeconds: 600
```

Comment:
as pointed by @celdrake , this 600 seconds is very low in order to get through all the steps to start device/VM. Maybe in this example (which most will just copy-paste), lets rise the value ?

Reply:
Good idea, updated

Reply:
Could you also please update the timeout in the `examples/csr.yaml` file?

I think it's convenient for using it as a template to create our own CSRs.

Reply:
Done